webpack.config.js

```diff
@@ -189,32 +189,59 @@ if (!IS_PRODUCTION && REFRESH) {
   );
 }
 
+/**
+ * Webpack entry for password strength checker
+ */
+var pwConfig = {
```

Comment:
Definitely agree, the storybook config also creates another entry point, so once everything gets merged in I'll work on refactoring the conf.